components/automate-cli/cmd/chef-automate/automateHAAwsDeployment.go

```diff
@@ -233,22 +233,22 @@ func (a *awsDeployment) validateEnvFields() *list.List {
 	if len(a.config.Aws.Config.ChefEbsVolumeType) < 1 {
 		errorList.PushBack("Invalid or empty aws chef_ebs_volume_type")
 	}
-	if len(a.config.Aws.Config.OpensearchEbsVolumeIops) < 1 {
+	if !a.config.Aws.Config.SetupManagedServices && len(a.config.Aws.Config.OpensearchEbsVolumeIops) < 1 {
```

Comment:
instead of adding the new condition to each and every if statement, we can write one if statement with this condition, and move the existing statements as nested.